components/membership/membership-persistence-client-impl/src/main/kotlin/net/corda/membership/impl/persistence/client/MembershipPersistenceOperationImpl.kt

```diff
@@ -0,0 +1,88 @@
+package net.corda.membership.impl.persistence.client
+
+import net.corda.data.membership.db.request.MembershipPersistenceRequest
+import net.corda.data.membership.db.request.async.MembershipPersistenceAsyncRequest
+import net.corda.data.membership.db.response.MembershipPersistenceResponse
+import net.corda.data.membership.db.response.query.PersistenceFailedResponse
+import net.corda.membership.persistence.client.MembershipPersistenceOperation
+import net.corda.membership.persistence.client.MembershipPersistenceResult
+import net.corda.messaging.api.publisher.RPCSender
+import net.corda.messaging.api.records.Record
+import net.corda.schema.Schemas.Membership.MEMBERSHIP_DB_ASYNC_TOPIC
+import net.corda.utilities.Either
+import net.corda.utilities.concurrent.getOrThrow
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.util.concurrent.TimeoutException
+
+/**
+ * An implementation of the MembershipPersistenceOperation.
+ *
+ * @param sender - An RPC sender to enable the send request.
+ * @param request The persistence request (to be sent either via the RPC sender or to
+ *   create the asynchronous command from).
+ * @param convertResult - A function to convert the result from the
```

Comment:
```suggestion
 * @param sender An RPC sender to enable the send request.
 * @param request The persistence request (to be sent either via the RPC sender or to
 *   create the asynchronous command from).
 * @param convertResult A function to convert the result from the
```
The dashes break the rendering